pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+skip-string-normalization = false
+include = '\.pyi?$'
+
+[tool.isort]
+profile = "black"                 # black-compatible (e.g., trailing comma)
+known_first_party = ["pbench"]    # separate "pbench" section
```

Comment:
This does not appear to apply.

Reply:
Fixed, I think.

Due to the fact that this project only has one source file (so it doesn't import any local sources), this change doesn't affect the main sources; this would matter for the unit test file, but, due to circumstances, it doesn't actually have an effect there, either...so I'm not _sure_ that it's fixed...but I changed it!  😀